styleguide/_themes/global/scss/grid.scss

```diff
@@ -1,10 +1,20 @@
+$max-row-width: 960px;
+
+@mixin rs-make-row($gutter: $grid-gutter-width) {
+  @include make-row($gutter);
+  margin: 0 auto;
+  max-width: $max-row-width;
+}
```

Comment:
Unfortunately I couldn't just write over the existing `make-row` mixin, so I had to create an alternative, that basically extends basic `make-row` functionality, but also centers it, and sets a maximum width. All existing references to `make-row` have thus been changed to `rs-make-row`